src/components/views/Basket/Basket.test.js

```diff
@@ -81,62 +79,6 @@ describe('Basket', () => {
     await Button('Create new agreement').exists();
   });
 
-  describe('Choosing an option and clicking the add to selected agreement button', () => {
```

Comment:
Removing a load of tests and not replacing them is not really acceptable. We should be replacing the functionality with a mock where possible to ensure that our module does the right thing when that mocked component (not agreementFilterButton) has inputs/outputs